exercises/generics/generics1.rs

```diff
@@ -6,9 +6,7 @@
 // Execute `rustlings hint generics1` or use the `hint` watch subcommand for a
 // hint.
 
-// I AM NOT DONE
-
 fn main() {
-    let mut shopping_list: Vec<?> = Vec::new();
+    let mut shopping_list: Vec<_> = Vec::new();
```

Comment:
Can you think of another way to modify this line?

Reply:
<T> can't work here since we need to define this T variable.
We can use a function that gets Vec<T>, push another item, and return a new Vec.
but in this case, we need to initiate some Vector in the main function.
Is there another way?

Reply:
`Vec<String>` nothing too fancy :)